README.md

```diff
@@ -232,7 +232,7 @@ Run `carthage update` and drag `SwiftProtobuf.framework` into your Xcode.project
 
 # Quick Start
 
-Once you have installed the code generator, used it to
+Once you have installed the code generator, use it to
 generate Swift code from your `.proto` file, and
 added the SwiftProtobuf library to your project, you can
```

Comment:
I think the intent was for this long sentence to be all past tense "installed", "used", "added".  So if you are shifting "used" to "use", then "added" should likely change to "add"?

Reply:
Yes, makes sense. Thanks for your review.